src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -62,9 +65,42 @@ public class Program
 }
 ```
 
+#### For Metrics
+
+The following example demonstrates adding HTTP client instrumentation with the
+extension method `.AddHttpClientInstrumentation()` to a console application.
+
+The metric and its attributes being implemented are following Opentelemetry
```

Comment:
please move this outside this subheading, to keep this section focused only on the enablement steps.